cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -30,100 +30,177 @@
-    'title': 'Money Topics',
-    'hide_title': 'true',
-    'nav_items': [
-        {'text': 'Money Transfers', 'url': '/sending-money/'},
-        {'text': 'Mortgages', 'url': '/consumer-tools/mortgages/'},
-        {'text': 'Payday Loans', 'url': '/ask-cfpb/category-payday-loans/'},
-        {'text': 'Prepaid Cards', 'url': '/consumer-tools/prepaid-cards/'},
-        {'text': 'Student Loans', 'url': '/consumer-tools/student-loans/'},
-    ]
+    'type': 'nav_group',
+    'value': {
+        'group_title': 'Money Topics',
+        'hide_group_title': 'true',
+        'nav_items': [
+            {'link':{
+                'link_text': 'Money Transfers',
+                'external_link': '/sending-money/'
+            }},
+            {'link':{
+                'link_text': 'Mortgages',
+                'external_link': '/consumer-tools/mortgages/'
+            }},
+            {'link':{
+                'link_text': 'Payday Loans',
+                'external_link': '/ask-cfpb/category-payday-loans/'
+            }},
+            {'link':{
+                'link_text': 'Prepaid Cards',
+                'external_link': '/consumer-tools/prepaid-cards/'
+            }},
+            {'link':{
+                'link_text': 'Student Loans',
+                'external_link': '/consumer-tools/student-loans/'
+            }},
+        ]
+    }
 } %}
 
 {% set consumer_tools_group_three = {
-    'title': 'Guides',
-    'nav_items': [
-        {'text': 'Getting an Auto Loan', 'url': '/consumer-tools/getting-an-auto-loan/'},
-        {'text': 'Managing Someone Else’s Money', 'url': '/consumer-tools/managing-someone-elses-money/'},
-        {'text': 'Money as You Grow', 'url': '/consumer-tools/money-as-you-grow/'},
-        {'text': 'Navigating the Military Financial Lifecycle', 'url': '/consumer-tools/military-financial-lifecycle/'},
-        {'text': 'Owning a Home', 'url': '/owning-a-home/'},
-        {'text': 'Paying for College', 'url': '/paying-for-college/'},
-        {'text': 'Planning for Retirement', 'url': '/consumer-tools/retirement/'},
-    ]
+    'type': 'nav_group',
+    'value': {
+        'group_title': 'Guides',
+        'nav_items': [
+            {'link':{
+                'link_text': 'Getting an Auto Loan',
+                'external_link': '/consumer-tools/getting-an-auto-loan/'
+            }},
+            {'link':{
+                'link_text': 'Managing Someone Else’s Money',
+                'external_link': '/consumer-tools/managing-someone-elses-money/'
+            }},
+            {'link':{
+                'link_text': 'Money as You Grow',
+                'external_link': '/consumer-tools/money-as-you-grow/'
+            }},
+            {'link':{
+                'link_text': 'Navigating the Military Financial Lifecycle',
+                'external_link': '/consumer-tools/military-financial-lifecycle/'
+            }},
+            {'link':{
+                'link_text': 'Owning a Home',
```

Comment:
Does it make sense to rename this to "Buying a House"? See #3732 and GHE#2390.

Reply:
Link text is updated in https://github.com/cfpb/cfgov-refresh/pull/3396/commits/9e021bad68cc8e75a4ea72d5ebd499b0566417d6